src/_posts/compliance/security-overview/2000-01-01-operational-security.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Operational Security
+nav: Operations
+modified_at: 2024-06-20 00:00:00
+tags: compliance security measures
+index: 8
+---
+
+### Logical Access Control
+
+Access to the platform administration is restricted to authorized personnel only. It is secured by two-factor
+authentication and when possible uses a bastion host and a VPN connection.
```

Comment:
Maybe too much details for a public document

Reply:
There is no actionnable information here